I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 138, in script
    "Kelas" "Baik, Bu."
  File "renpy/common/00library.rpy", line 268, in say
    who(what, interact=interact, *args, **kwargs)
IOError: Couldn't find file 'audio/book.mp3'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 138, in script
    "Kelas" "Baik, Bu."
  File "/Users/owner/pypypy/renpy/renpy-7.4.11-sdk/renpy/ast.py", line 721, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "/Users/owner/pypypy/renpy/renpy-7.4.11-sdk/renpy/exports.py", line 1417, in say
    renpy.store.say(who, what, *args, **kwargs)
  File "renpy/common/00library.rpy", line 268, in say
    who(what, interact=interact, *args, **kwargs)
  File "/Users/owner/pypypy/renpy/renpy-7.4.11-sdk/renpy/character.py", line 1249, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "/Users/owner/pypypy/renpy/renpy-7.4.11-sdk/renpy/character.py", line 901, in do_display
    **display_args)
  File "/Users/owner/pypypy/renpy/renpy-7.4.11-sdk/renpy/character.py", line 647, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "/Users/owner/pypypy/renpy/renpy-7.4.11-sdk/renpy/ui.py", line 298, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Users/owner/pypypy/renpy/renpy-7.4.11-sdk/renpy/display/core.py", line 3325, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, **kwargs)
  File "/Users/owner/pypypy/renpy/renpy-7.4.11-sdk/renpy/display/core.py", line 4025, in interact_core
    renpy.audio.audio.periodic()
  File "/Users/owner/pypypy/renpy/renpy-7.4.11-sdk/renpy/audio/audio.py", line 1162, in periodic
    raise_(exc[0], exc[1], exc[2])
  File "/Users/owner/pypypy/renpy/renpy-7.4.11-sdk/renpy/audio/audio.py", line 1140, in periodic_thread_main
    periodic_pass()
  File "/Users/owner/pypypy/renpy/renpy-7.4.11-sdk/renpy/audio/audio.py", line 1082, in periodic_pass
    c.periodic()
  File "/Users/owner/pypypy/renpy/renpy-7.4.11-sdk/renpy/audio/audio.py", line 518, in periodic
    topf = load(filename)
  File "/Users/owner/pypypy/renpy/renpy-7.4.11-sdk/renpy/audio/audio.py", line 69, in load
    rv = renpy.loader.load(fn)
  File "/Users/owner/pypypy/renpy/renpy-7.4.11-sdk/renpy/loader.py", line 798, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'audio/book.mp3'.

Darwin-18.7.0-x86_64-i386-64bit
Ren'Py 7.4.11.2266
EduGame 1.0
Fri Apr 15 04:55:00 2022
